### PR TITLE
bug fix: prepareEditorContentNode node id was not unique for each field

### PR DESCRIPTION
**Pull request by asa511**

### CODE REVIEW

File: src/helpers/normalize.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ const compose = require('compose-then')
 const { result, isPlainObject, curry, get } = require('lodash')
 const pascalCase = require('pascalcase')
 const { downloadEntryImages } = require('./images')
+const { logInfo, logError } = require('./logger')
```
Comment (jperasmus) [inline src/helpers/normalize.js:5]: Hi, thanks so much for your contribution!

You can remove this import again if you're not using it.

File: src/helpers/normalize.js [hunk 2]
```diff
@@ -225,10 +228,12 @@ const processContentEntry = async ({ schema, locale, entry, gatsbyHelpers }) =>
       fieldType,
       editorContent,
       nodeId,
+      index,
       gatsbyHelpers
     })
 
     childrenNodes.push(contentNode.id)
+    index = index + 1
```
Comment (jperasmus) [inline src/helpers/normalize.js:236]: Changing this to `index += 1` should resolve the linting error.

Comment (asa511) [reply src/helpers/normalize.js:236]: I see that this issue is fixed in the develop branch. This merge request can be rejected.

File: src/helpers/normalize.js (hunk 1 above)
Comment (asa511) [reply src/helpers/normalize.js:5]: I have withdrawn the merge request since this issue has been fixed in the develop branch already.

Comment (jperasmus) [reply src/helpers/normalize.js:5]: Okay, I'm not sure which version you are currently using, but if you want to get the latest code, you can install with the `next` tag.

**Pull request closed without merging**